client/styles/Home.module.css

```diff
@@ -0,0 +1,121 @@
+.container {
+  min-height: 100vh;
+  padding: 0 0.5rem;
+  display: flex;
```

Comment:
Separate out commonly used styles and group them.

Use styled components to their full extent - extending css, existing styled components, etc.